### ISSUE TITLE
reesmat.tst broken by Semigroups package

### ISSUE THREAD

**Issue by alex-konovalov** [gap-system/gap#258]
After the most recent update of Semigroups 2.6 by @james-d-mitchell, picked up on September 23rd, the following two diffs appear in the test when all packages are loaded:
```
########> Diff in /mnt/disk2/hudson-slave/workspace/GAP-stable/GAPCOPTS/64buil\
d/GAPGMP/nogmp/GAPTARGET/standard/label/64bit/GAP-git-stable-snapshot/tst/rees\
mat.tst, line 92:
# Input is:
Elements(V);
# Expected output:
[ 0, (3,(),1), (3,(),3), (3,(),8), (3,(),9), (3,(1,2)(3,5)(4,6),1), 
  (3,(1,2)(3,5)(4,6),3), (3,(1,2)(3,5)(4,6),8), (3,(1,2)(3,5)(4,6),9), 
  (3,(1,3)(2,4)(5,6),1), (3,(1,3)(2,4)(5,6),3), (3,(1,3)(2,4)(5,6),8), 
  (3,(1,3)(2,4)(5,6),9), (3,(1,4,5)(2,3,6),1), (3,(1,4,5)(2,3,6),3), 
  (3,(1,4,5)(2,3,6),8), (3,(1,4,5)(2,3,6),9), (3,(1,5,4)(2,6,3),1), 
  (3,(1,5,4)(2,6,3),3), (3,(1,5,4)(2,6,3),8), (3,(1,5,4)(2,6,3),9), 
  (3,(1,6)(2,5)(3,4),1), (3,(1,6)(2,5)(3,4),3), (3,(1,6)(2,5)(3,4),8), 
  (3,(1,6)(2,5)(3,4),9), (11,(),1), (11,(),3), (11,(),8), (11,(),9), 
  (11,(1,2)(3,5)(4,6),1), (11,(1,2)(3,5)(4,6),3), (11,(1,2)(3,5)(4,6),8), 
  (11,(1,2)(3,5)(4,6),9), (11,(1,3)(2,4)(5,6),1), (11,(1,3)(2,4)(5,6),3), 
  (11,(1,3)(2,4)(5,6),8), (11,(1,3)(2,4)(5,6),9), (11,(1,4,5)(2,3,6),1), 
  (11,(1,4,5)(2,3,6),3), (11,(1,4,5)(2,3,6),8), (11,(1,4,5)(2,3,6),9), 
  (11,(1,5,4)(2,6,3),1), (11,(1,5,4)(2,6,3),3), (11,(1,5,4)(2,6,3),8), 
  (11,(1,5,4)(2,6,3),9), (11,(1,6)(2,5)(3,4),1), (11,(1,6)(2,5)(3,4),3), 
  (11,(1,6)(2,5)(3,4),8), (11,(1,6)(2,5)(3,4),9), (13,(),1), (13,(),3), 
  (13,(),8), (13,(),9), (13,(1,2)(3,5)(4,6),1), (13,(1,2)(3,5)(4,6),3), 
  (13,(1,2)(3,5)(4,6),8), (13,(1,2)(3,5)(4,6),9), (13,(1,3)(2,4)(5,6),1), 
  (13,(1,3)(2,4)(5,6),3), (13,(1,3)(2,4)(5,6),8), (13,(1,3)(2,4)(5,6),9), 
  (13,(1,4,5)(2,3,6),1), (13,(1,4,5)(2,3,6),3), (13,(1,4,5)(2,3,6),8), 
  (13,(1,4,5)(2,3,6),9), (13,(1,5,4)(2,6,3),1), (13,(1,5,4)(2,6,3),3), 
  (13,(1,5,4)(2,6,3),8), (13,(1,5,4)(2,6,3),9), (13,(1,6)(2,5)(3,4),1), 
  (13,(1,6)(2,5)(3,4),3), (13,(1,6)(2,5)(3,4),8), (13,(1,6)(2,5)(3,4),9), 
  (21,(),1), (21,(),3), (21,(),8), (21,(),9), (21,(1,2)(3,5)(4,6),1), 
  (21,(1,2)(3,5)(4,6),3), (21,(1,2)(3,5)(4,6),8), (21,(1,2)(3,5)(4,6),9), 
  (21,(1,3)(2,4)(5,6),1), (21,(1,3)(2,4)(5,6),3), (21,(1,3)(2,4)(5,6),8), 
  (21,(1,3)(2,4)(5,6),9), (21,(1,4,5)(2,3,6),1), (21,(1,4,5)(2,3,6),3), 
  (21,(1,4,5)(2,3,6),8), (21,(1,4,5)(2,3,6),9), (21,(1,5,4)(2,6,3),1), 
  (21,(1,5,4)(2,6,3),3), (21,(1,5,4)(2,6,3),8), (21,(1,5,4)(2,6,3),9), 
  (21,(1,6)(2,5)(3,4),1), (21,(1,6)(2,5)(3,4),3), (21,(1,6)(2,5)(3,4),8), 
  (21,(1,6)(2,5)(3,4),9) ]
# But found:
[ 0, (3,(),1), (3,(),3), (3,(),8), (3,(),9), (3,(1,2)(3,5)(4,6),1), (3,(1,2)
  (3,5)(4,6),3), (3,(1,2)(3,5)(4,6),8), (3,(1,2)(3,5)(4,6),9), (3,(1,3)(2,4)
  (5,6),1), (3,(1,3)(2,4)(5,6),3), (3,(1,3)(2,4)(5,6),8), (3,(1,3)(2,4)
  (5,6),9), (3,(1,4,5)(2,3,6),1), (3,(1,4,5)(2,3,6),3), (3,(1,4,5)(2,3,6),8), 
  (3,(1,4,5)(2,3,6),9), (3,(1,5,4)(2,6,3),1), (3,(1,5,4)(2,6,3),3), (3,(1,5,4)
  (2,6,3),8), (3,(1,5,4)(2,6,3),9), (3,(1,6)(2,5)(3,4),1), (3,(1,6)(2,5)
  (3,4),3), (3,(1,6)(2,5)(3,4),8), (3,(1,6)(2,5)(3,4),9), (11,(),1), 
  (11,(),3), (11,(),8), (11,(),9), (11,(1,2)(3,5)(4,6),1), (11,(1,2)(3,5)
  (4,6),3), (11,(1,2)(3,5)(4,6),8), (11,(1,2)(3,5)(4,6),9), (11,(1,3)(2,4)
  (5,6),1), (11,(1,3)(2,4)(5,6),3), (11,(1,3)(2,4)(5,6),8), (11,(1,3)(2,4)
  (5,6),9), (11,(1,4,5)(2,3,6),1), (11,(1,4,5)(2,3,6),3), (11,(1,4,5)
  (2,3,6),8), (11,(1,4,5)(2,3,6),9), (11,(1,5,4)(2,6,3),1), (11,(1,5,4)
  (2,6,3),3), (11,(1,5,4)(2,6,3),8), (11,(1,5,4)(2,6,3),9), (11,(1,6)(2,5)
  (3,4),1), (11,(1,6)(2,5)(3,4),3), (11,(1,6)(2,5)(3,4),8), (11,(1,6)(2,5)
  (3,4),9), (13,(),1), (13,(),3), (13,(),8), (13,(),9), (13,(1,2)(3,5)
  (4,6),1), (13,(1,2)(3,5)(4,6),3), (13,(1,2)(3,5)(4,6),8), (13,(1,2)(3,5)
  (4,6),9), (13,(1,3)(2,4)(5,6),1), (13,(1,3)(2,4)(5,6),3), (13,(1,3)(2,4)
  (5,6),8), (13,(1,3)(2,4)(5,6),9), (13,(1,4,5)(2,3,6),1), (13,(1,4,5)
  (2,3,6),3), (13,(1,4,5)(2,3,6),8), (13,(1,4,5)(2,3,6),9), (13,(1,5,4)
  (2,6,3),1), (13,(1,5,4)(2,6,3),3), (13,(1,5,4)(2,6,3),8), (13,(1,5,4)
  (2,6,3),9), (13,(1,6)(2,5)(3,4),1), (13,(1,6)(2,5)(3,4),3), (13,(1,6)(2,5)
  (3,4),8), (13,(1,6)(2,5)(3,4),9), (21,(),1), (21,(),3), (21,(),8), 
  (21,(),9), (21,(1,2)(3,5)(4,6),1), (21,(1,2)(3,5)(4,6),3), (21,(1,2)(3,5)
  (4,6),8), (21,(1,2)(3,5)(4,6),9), (21,(1,3)(2,4)(5,6),1), (21,(1,3)(2,4)
  (5,6),3), (21,(1,3)(2,4)(5,6),8), (21,(1,3)(2,4)(5,6),9), (21,(1,4,5)
  (2,3,6),1), (21,(1,4,5)(2,3,6),3), (21,(1,4,5)(2,3,6),8), (21,(1,4,5)
  (2,3,6),9), (21,(1,5,4)(2,6,3),1), (21,(1,5,4)(2,6,3),3), (21,(1,5,4)
  (2,6,3),8), (21,(1,5,4)(2,6,3),9), (21,(1,6)(2,5)(3,4),1), (21,(1,6)(2,5)
  (3,4),3), (21,(1,6)(2,5)(3,4),8), (21,(1,6)(2,5)(3,4),9) ]
########
########> Diff in /mnt/disk2/hudson-slave/workspace/GAP-stable/GAPCOPTS/64buil\
d/GAPGMP/nogmp/GAPTARGET/standard/label/64bit/GAP-git-stable-snapshot/tst/rees\
mat.tst, line 432:
# Input is:
Elements(V);
# Expected output:
[ (2,(),2), (2,(),4), (2,(),11), (2,(2,8)(3,7)(4,6),2), (2,(2,8)(3,7)(4,6),4),
  (2,(2,8)(3,7)(4,6),11), (2,(1,2)(3,8)(4,7)(5,6),2), 
  (2,(1,2)(3,8)(4,7)(5,6),4), (2,(1,2)(3,8)(4,7)(5,6),11), 
  (2,(1,2,3,4,5,6,7,8),2), (2,(1,2,3,4,5,6,7,8),4), (2,(1,2,3,4,5,6,7,8),11), 
  (2,(1,3)(4,8)(5,7),2), (2,(1,3)(4,8)(5,7),4), (2,(1,3)(4,8)(5,7),11), 
  (2,(1,3,5,7)(2,4,6,8),2), (2,(1,3,5,7)(2,4,6,8),4), 
  (2,(1,3,5,7)(2,4,6,8),11), (2,(1,4)(2,3)(5,8)(6,7),2), 
  (2,(1,4)(2,3)(5,8)(6,7),4), (2,(1,4)(2,3)(5,8)(6,7),11), 
  (2,(1,4,7,2,5,8,3,6),2), (2,(1,4,7,2,5,8,3,6),4), (2,(1,4,7,2,5,8,3,6),11), 
  (2,(1,5)(2,4)(6,8),2), (2,(1,5)(2,4)(6,8),4), (2,(1,5)(2,4)(6,8),11), 
  (2,(1,5)(2,6)(3,7)(4,8),2), (2,(1,5)(2,6)(3,7)(4,8),4), 
  (2,(1,5)(2,6)(3,7)(4,8),11), (2,(1,6)(2,5)(3,4)(7,8),2), 
  (2,(1,6)(2,5)(3,4)(7,8),4), (2,(1,6)(2,5)(3,4)(7,8),11), 
  (2,(1,6,3,8,5,2,7,4),2), (2,(1,6,3,8,5,2,7,4),4), (2,(1,6,3,8,5,2,7,4),11), 
  (2,(1,7)(2,6)(3,5),2), (2,(1,7)(2,6)(3,5),4), (2,(1,7)(2,6)(3,5),11), 
  (2,(1,7,5,3)(2,8,6,4),2), (2,(1,7,5,3)(2,8,6,4),4), 
  (2,(1,7,5,3)(2,8,6,4),11), (2,(1,8,7,6,5,4,3,2),2), (2,(1,8,7,6,5,4,3,2),4),
  (2,(1,8,7,6,5,4,3,2),11), (2,(1,8)(2,7)(3,6)(4,5),2), 
  (2,(1,8)(2,7)(3,6)(4,5),4), (2,(1,8)(2,7)(3,6)(4,5),11), (6,(),2), 
  (6,(),4), (6,(),11), (6,(2,8)(3,7)(4,6),2), (6,(2,8)(3,7)(4,6),4), 
  (6,(2,8)(3,7)(4,6),11), (6,(1,2)(3,8)(4,7)(5,6),2), 
  (6,(1,2)(3,8)(4,7)(5,6),4), (6,(1,2)(3,8)(4,7)(5,6),11), 
  (6,(1,2,3,4,5,6,7,8),2), (6,(1,2,3,4,5,6,7,8),4), (6,(1,2,3,4,5,6,7,8),11), 
  (6,(1,3)(4,8)(5,7),2), (6,(1,3)(4,8)(5,7),4), (6,(1,3)(4,8)(5,7),11), 
  (6,(1,3,5,7)(2,4,6,8),2), (6,(1,3,5,7)(2,4,6,8),4), 
  (6,(1,3,5,7)(2,4,6,8),11), (6,(1,4)(2,3)(5,8)(6,7),2), 
  (6,(1,4)(2,3)(5,8)(6,7),4), (6,(1,4)(2,3)(5,8)(6,7),11), 
  (6,(1,4,7,2,5,8,3,6),2), (6,(1,4,7,2,5,8,3,6),4), (6,(1,4,7,2,5,8,3,6),11), 
  (6,(1,5)(2,4)(6,8),2), (6,(1,5)(2,4)(6,8),4), (6,(1,5)(2,4)(6,8),11), 
  (6,(1,5)(2,6)(3,7)(4,8),2), (6,(1,5)(2,6)(3,7)(4,8),4), 
  (6,(1,5)(2,6)(3,7)(4,8),11), (6,(1,6)(2,5)(3,4)(7,8),2), 
  (6,(1,6)(2,5)(3,4)(7,8),4), (6,(1,6)(2,5)(3,4)(7,8),11), 
  (6,(1,6,3,8,5,2,7,4),2), (6,(1,6,3,8,5,2,7,4),4), (6,(1,6,3,8,5,2,7,4),11), 
  (6,(1,7)(2,6)(3,5),2), (6,(1,7)(2,6)(3,5),4), (6,(1,7)(2,6)(3,5),11), 
  (6,(1,7,5,3)(2,8,6,4),2), (6,(1,7,5,3)(2,8,6,4),4), 
  (6,(1,7,5,3)(2,8,6,4),11), (6,(1,8,7,6,5,4,3,2),2), (6,(1,8,7,6,5,4,3,2),4),
  (6,(1,8,7,6,5,4,3,2),11), (6,(1,8)(2,7)(3,6)(4,5),2), 
  (6,(1,8)(2,7)(3,6)(4,5),4), (6,(1,8)(2,7)(3,6)(4,5),11), (9,(),2), 
  (9,(),4), (9,(),11), (9,(2,8)(3,7)(4,6),2), (9,(2,8)(3,7)(4,6),4), 
  (9,(2,8)(3,7)(4,6),11), (9,(1,2)(3,8)(4,7)(5,6),2), 
  (9,(1,2)(3,8)(4,7)(5,6),4), (9,(1,2)(3,8)(4,7)(5,6),11), 
  (9,(1,2,3,4,5,6,7,8),2), (9,(1,2,3,4,5,6,7,8),4), (9,(1,2,3,4,5,6,7,8),11), 
  (9,(1,3)(4,8)(5,7),2), (9,(1,3)(4,8)(5,7),4), (9,(1,3)(4,8)(5,7),11), 
  (9,(1,3,5,7)(2,4,6,8),2), (9,(1,3,5,7)(2,4,6,8),4), 
  (9,(1,3,5,7)(2,4,6,8),11), (9,(1,4)(2,3)(5,8)(6,7),2), 
  (9,(1,4)(2,3)(5,8)(6,7),4), (9,(1,4)(2,3)(5,8)(6,7),11), 
  (9,(1,4,7,2,5,8,3,6),2), (9,(1,4,7,2,5,8,3,6),4), (9,(1,4,7,2,5,8,3,6),11), 
  (9,(1,5)(2,4)(6,8),2), (9,(1,5)(2,4)(6,8),4), (9,(1,5)(2,4)(6,8),11), 
  (9,(1,5)(2,6)(3,7)(4,8),2), (9,(1,5)(2,6)(3,7)(4,8),4), 
  (9,(1,5)(2,6)(3,7)(4,8),11), (9,(1,6)(2,5)(3,4)(7,8),2), 
  (9,(1,6)(2,5)(3,4)(7,8),4), (9,(1,6)(2,5)(3,4)(7,8),11), 
  (9,(1,6,3,8,5,2,7,4),2), (9,(1,6,3,8,5,2,7,4),4), (9,(1,6,3,8,5,2,7,4),11), 
  (9,(1,7)(2,6)(3,5),2), (9,(1,7)(2,6)(3,5),4), (9,(1,7)(2,6)(3,5),11), 
  (9,(1,7,5,3)(2,8,6,4),2), (9,(1,7,5,3)(2,8,6,4),4), 
  (9,(1,7,5,3)(2,8,6,4),11), (9,(1,8,7,6,5,4,3,2),2), (9,(1,8,7,6,5,4,3,2),4),
  (9,(1,8,7,6,5,4,3,2),11), (9,(1,8)(2,7)(3,6)(4,5),2), 
  (9,(1,8)(2,7)(3,6)(4,5),4), (9,(1,8)(2,7)(3,6)(4,5),11) ]
# But found:
[ (2,(),2), (2,(),4), (2,(),11), (2,(2,8)(3,7)(4,6),2), (2,(2,8)(3,7)
  (4,6),4), (2,(2,8)(3,7)(4,6),11), (2,(1,2)(3,8)(4,7)(5,6),2), (2,(1,2)(3,8)
  (4,7)(5,6),4), (2,(1,2)(3,8)(4,7)(5,6),11), (2,(1,2,3,4,5,6,7,8),2), 
  (2,(1,2,3,4,5,6,7,8),4), (2,(1,2,3,4,5,6,7,8),11), (2,(1,3)(4,8)(5,7),2), 
  (2,(1,3)(4,8)(5,7),4), (2,(1,3)(4,8)(5,7),11), (2,(1,3,5,7)(2,4,6,8),2), 
  (2,(1,3,5,7)(2,4,6,8),4), (2,(1,3,5,7)(2,4,6,8),11), (2,(1,4)(2,3)(5,8)
  (6,7),2), (2,(1,4)(2,3)(5,8)(6,7),4), (2,(1,4)(2,3)(5,8)(6,7),11), 
  (2,(1,4,7,2,5,8,3,6),2), (2,(1,4,7,2,5,8,3,6),4), (2,(1,4,7,2,5,8,3,6),11), 
  (2,(1,5)(2,4)(6,8),2), (2,(1,5)(2,4)(6,8),4), (2,(1,5)(2,4)(6,8),11), 
  (2,(1,5)(2,6)(3,7)(4,8),2), (2,(1,5)(2,6)(3,7)(4,8),4), (2,(1,5)(2,6)(3,7)
  (4,8),11), (2,(1,6)(2,5)(3,4)(7,8),2), (2,(1,6)(2,5)(3,4)(7,8),4), (2,(1,6)
  (2,5)(3,4)(7,8),11), (2,(1,6,3,8,5,2,7,4),2), (2,(1,6,3,8,5,2,7,4),4), 
  (2,(1,6,3,8,5,2,7,4),11), (2,(1,7)(2,6)(3,5),2), (2,(1,7)(2,6)(3,5),4), 
  (2,(1,7)(2,6)(3,5),11), (2,(1,7,5,3)(2,8,6,4),2), (2,(1,7,5,3)(2,8,6,4),4), 
  (2,(1,7,5,3)(2,8,6,4),11), (2,(1,8,7,6,5,4,3,2),2), (2,(1,8,7,6,5,4,3,2),4),
  (2,(1,8,7,6,5,4,3,2),11), (2,(1,8)(2,7)(3,6)(4,5),2), (2,(1,8)(2,7)(3,6)
  (4,5),4), (2,(1,8)(2,7)(3,6)(4,5),11), (6,(),2), (6,(),4), (6,(),11), 
  (6,(2,8)(3,7)(4,6),2), (6,(2,8)(3,7)(4,6),4), (6,(2,8)(3,7)(4,6),11), 
  (6,(1,2)(3,8)(4,7)(5,6),2), (6,(1,2)(3,8)(4,7)(5,6),4), (6,(1,2)(3,8)(4,7)
  (5,6),11), (6,(1,2,3,4,5,6,7,8),2), (6,(1,2,3,4,5,6,7,8),4), 
  (6,(1,2,3,4,5,6,7,8),11), (6,(1,3)(4,8)(5,7),2), (6,(1,3)(4,8)(5,7),4), 
  (6,(1,3)(4,8)(5,7),11), (6,(1,3,5,7)(2,4,6,8),2), (6,(1,3,5,7)(2,4,6,8),4), 
  (6,(1,3,5,7)(2,4,6,8),11), (6,(1,4)(2,3)(5,8)(6,7),2), (6,(1,4)(2,3)(5,8)
  (6,7),4), (6,(1,4)(2,3)(5,8)(6,7),11), (6,(1,4,7,2,5,8,3,6),2), 
  (6,(1,4,7,2,5,8,3,6),4), (6,(1,4,7,2,5,8,3,6),11), (6,(1,5)(2,4)(6,8),2), 
  (6,(1,5)(2,4)(6,8),4), (6,(1,5)(2,4)(6,8),11), (6,(1,5)(2,6)(3,7)(4,8),2), 
  (6,(1,5)(2,6)(3,7)(4,8),4), (6,(1,5)(2,6)(3,7)(4,8),11), (6,(1,6)(2,5)(3,4)
  (7,8),2), (6,(1,6)(2,5)(3,4)(7,8),4), (6,(1,6)(2,5)(3,4)(7,8),11), 
  (6,(1,6,3,8,5,2,7,4),2), (6,(1,6,3,8,5,2,7,4),4), (6,(1,6,3,8,5,2,7,4),11), 
  (6,(1,7)(2,6)(3,5),2), (6,(1,7)(2,6)(3,5),4), (6,(1,7)(2,6)(3,5),11), 
  (6,(1,7,5,3)(2,8,6,4),2), (6,(1,7,5,3)(2,8,6,4),4), (6,(1,7,5,3)
  (2,8,6,4),11), (6,(1,8,7,6,5,4,3,2),2), (6,(1,8,7,6,5,4,3,2),4), 
  (6,(1,8,7,6,5,4,3,2),11), (6,(1,8)(2,7)(3,6)(4,5),2), (6,(1,8)(2,7)(3,6)
  (4,5),4), (6,(1,8)(2,7)(3,6)(4,5),11), (9,(),2), (9,(),4), (9,(),11), 
  (9,(2,8)(3,7)(4,6),2), (9,(2,8)(3,7)(4,6),4), (9,(2,8)(3,7)(4,6),11), 
  (9,(1,2)(3,8)(4,7)(5,6),2), (9,(1,2)(3,8)(4,7)(5,6),4), (9,(1,2)(3,8)(4,7)
  (5,6),11), (9,(1,2,3,4,5,6,7,8),2), (9,(1,2,3,4,5,6,7,8),4), 
  (9,(1,2,3,4,5,6,7,8),11), (9,(1,3)(4,8)(5,7),2), (9,(1,3)(4,8)(5,7),4), 
  (9,(1,3)(4,8)(5,7),11), (9,(1,3,5,7)(2,4,6,8),2), (9,(1,3,5,7)(2,4,6,8),4), 
  (9,(1,3,5,7)(2,4,6,8),11), (9,(1,4)(2,3)(5,8)(6,7),2), (9,(1,4)(2,3)(5,8)
  (6,7),4), (9,(1,4)(2,3)(5,8)(6,7),11), (9,(1,4,7,2,5,8,3,6),2), 
  (9,(1,4,7,2,5,8,3,6),4), (9,(1,4,7,2,5,8,3,6),11), (9,(1,5)(2,4)(6,8),2), 
  (9,(1,5)(2,4)(6,8),4), (9,(1,5)(2,4)(6,8),11), (9,(1,5)(2,6)(3,7)(4,8),2), 
  (9,(1,5)(2,6)(3,7)(4,8),4), (9,(1,5)(2,6)(3,7)(4,8),11), (9,(1,6)(2,5)(3,4)
  (7,8),2), (9,(1,6)(2,5)(3,4)(7,8),4), (9,(1,6)(2,5)(3,4)(7,8),11), 
  (9,(1,6,3,8,5,2,7,4),2), (9,(1,6,3,8,5,2,7,4),4), (9,(1,6,3,8,5,2,7,4),11), 
  (9,(1,7)(2,6)(3,5),2), (9,(1,7)(2,6)(3,5),4), (9,(1,7)(2,6)(3,5),11), 
  (9,(1,7,5,3)(2,8,6,4),2), (9,(1,7,5,3)(2,8,6,4),4), (9,(1,7,5,3)
  (2,8,6,4),11), (9,(1,8,7,6,5,4,3,2),2), (9,(1,8,7,6,5,4,3,2),4), 
  (9,(1,8,7,6,5,4,3,2),11), (9,(1,8)(2,7)(3,6)(4,5),2), (9,(1,8)(2,7)(3,6)
  (4,5),4), (9,(1,8)(2,7)(3,6)(4,5),11) ]
########
```

**Comment by ChrisJefferson** [comment]
Assuming we don't care about this ordering (Hopefully @james-d-mitchell can confirm that), perhaps we should change the test to ```SortedList(Elements(V))```, which will make it more resistant to future changes (in general, I think using SortedList is a good idea when we notices tests produce things in an order we don't care about, and can change over time).

**Comment by alex-konovalov** [comment]
@ChrisJefferson: I first was going to suggest the same, but comparing initial lines, it looks to me that this about line breaking hints and not about the ordering (which seems to be the same). It's not a very good practice for a package to change the output of objects defined in the core GAP system...

**Comment by alex-konovalov** [comment]
@james-d-mitchell: FYI, another changed behaviour is now in my Circle package: 
```
########> Diff in /mnt/disk2/hudson-slave/workspace/GAP-minor-release-test/GAP\
COPTS/64build/GAPGMP/gmp/GAPTARGET/packages/label/64bit/gap4r7/pkg/circle/tst/\
circle03.tst, line 196:
# Input is:
H := GreensHClassOfElement(S,One(S));
# Expected output:
{CircleObject( ZmodnZObj( 0, 256 ) )}
# But found:
<Green's H-class: <object>>
```
and it happens only in the test with all packages loaded.

For my packages, I require that tests should run without diffs only with packages loaded by default, so I may tolerate this diff (maybe adjust the test to not to report it more).

**Comment by james-d-mitchell** [comment]
I agree with you Alex, it isn't very good practice for a package to change the output of objects defined in the core GAP system... The first diff (with Rees matrix semigroup elements) was unintentional. 

The other one (for H-classes) was intentional. I wanted a more meaningful display for Green's classes. It probably uses `ViewString` for your `CircleObject`.

I think what I should do is: make these changes in the GAP library and then make a pull request. And also make a new release of Semigroups. Does that sound ok?

**Comment by alex-konovalov** [comment]
@james-d-mitchell yes, that sounds OK, thanks - let's see how it goes with #262

**Comment by alex-konovalov** [comment]
@james-d-mitchell: in the new nightly Jenkins tests, `reesmat.tst` is still broken if the Semigroups package is loaded (so it's an issue for Semigroups, not for the core system).

I have an explanation why this is not seen with Travis: in that setting, we test core GAP system with the combination of packages that was included in the latest official release, that is at the moment GAP 4.7.8 (June 2015).

**Comment by alex-konovalov** [comment]
This is an issue with Semigroups package which should be fixed in Semigroups 2.7 by @james-d-mitchell (coming soon). For now, I am using the current stable version of Semigroups 2.5 for wrapping and testing release candidates.
